packages/doc/content/components/helpers/media.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Media'
+metaTitle: 'Media'
+metaDescription: 'Functions to help you dealing with responsive design'
+---
+
+import { Link } from 'gatsby';
+
+## Media
+
+The `media` module makes easy to write media queries using `styled-components`.
+
+It has a helper for each breakpoint, defined in our <Link to="/guidelines/tokens/breakpoints">guidelines</Link>:
+`xxs`, `xs`, `sm`, `md`, `lg`, `xl`, `xxl`, and `xxxl`.
+
+### Installing
+
+```bash type=highlight
+yarn add @gympass/yoga-helpers
+
+// or
+
+npm install @gympass/yoga-helpers
+```
+
+### Usage
+
+```javascript type=highlight
+import styled from 'styled-components';
+import { media } from '@gympass/yoga-helper';
+​
+const MyResponsiveButton = styled.button`
+  // default min
+  // @media (min-width: breakpoint)
+  ${media.md``}
+​
+  // you can also use as media.min(breakpoint)
+  // @media (min-width: breakpoint)
+  ${media.min('md')``}
+
+  // or media.max('breakpoint')
+  // @media (max-width: breakpoint)
+  ${media.max('xl')``}
+​
+  // for a device range
+  // @media (min-width: breakpoint) and (max-width: breakpoint)
+  ${media.between('md', 'l')``}
+​
+  // jest like
+  // @media not all and (min-width: breakpoint)
+  ${media.not.md``}
+
+  // @media not all and (max-width: breakpoint)
+  ${media.not.max('xxl')``}
+
+  // @media not all and (min-width: breakpoint) and (max-width: breakpoint)
+  ${media.not.between('sm', 'md')``}
+`;
+
+```
+
+The hide helper follows what we define in the <Link to="components/grid#hide">Hide</Link>
+component
+
+```javascript type=highlight
+import styled from 'styled-components';
+import { media } from '@gympass/yoga-helper';
+
+const HideMe = styled.div`
+  // will hide for, and only for, xxs
+  ${media.hide.xxs}
+
+  // or
+  // you can define when it will start hiding
+  ${media.hide['md-start']}
+`;
+
+const NotHideMe = styled.div`
+  // will hide for everyone besides sm
+  ${media.not.hide.sm}
+`;
+```
+
+### Example
```

Comment:
Do we need both `Usage` and `Example` sections? In my opinion, we could have only the `Example` section but name it as `Usage` 😛 

What do you guys think?

Reply:
I did what you suggested and it looks strange to me. I miss this section break between _what is available for you to use_ and _how to use it_.

In `Usage` I wanted to show the `media` API whereas the `Example` has a real instance of what could be used in a real-world application. 

It would be better to rename `Usage` to `API` and keep `Installing`, `API`, and `Usage`or just `Installing` and `Usage`?

Reply:
I like the ideia of having a separate section for the API and how to use it. My comment was based on what `styled-components` [does on its documentation](https://styled-components.com/docs/api#css).

If we want to showcase the API "possibilities" we could add a simpler segment with only the Object options, not necessarily inside a styled-component. For example:

```js
media[breakpoint]``
// Ex.: media.md`` or media['md-start']``

media.min(breakpoint)``
media.max(breakpoint)``
// Ex.: media.min('md')``

media.between(minBreakpoint, maxBreakpoint)``
// Ex.: media.between('md', 'l')``


// We can also use .not (Like jest expect matchers)
media.not[breakpoint]``

media.not.min(breakpoint)``
media.not.max(breakpoint)``

media.not.between(minBreakpoint, maxBreakpoint)``
```

So we would keep: `Installing`, `API`, and `Usage`. Would that have all the information the developer needs to use the helper?

Reply:
Much better!